### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by rafaelmaeuer**

### CODE REVIEW

File: openapi.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
       "name": "European Union Public License 1.2",
       "url": "https://spdx.org/licenses/EUPL-1.2.html"
     },
-    "version": "1.0.0-beta2"
+    "version": "1.0.0"
```
Comment (frytg) [inline openapi.json:14]: not `1.0.1`?

**Review by frytg — COMMENTED** [review]
not sure if `1.0.0` was intentional or whether we wanted `1.0.1`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.0] - 2021-05-05
```
Comment (rafaelmaeuer) [inline CHANGELOG.md:8]:
```suggestion
## [1.0.1] - 2021-05-05
```

File: openapi.json (hunk 1 above)
Comment (rafaelmaeuer) [inline openapi.json:14]:
```suggestion
    "version": "1.0.1"
```

File: openapi.yaml [hunk 3]
```diff
@@ -10,7 +10,7 @@ info:
   license:
     name: European Union Public License 1.2
     url: 'https://spdx.org/licenses/EUPL-1.2.html'
-  version: 1.0.0-beta2
+  version: 1.0.0
```
Comment (rafaelmaeuer) [inline openapi.yaml:13]:
```suggestion
  version: 1.0.1
```

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "ard-eventhub",
-	"version": "1.0.0-beta2",
+	"version": "1.0.0",
```
Comment (rafaelmaeuer) [inline package.json:3]:
```suggestion
	"version": "1.0.1",
```

**Review by frytg — APPROVED** [review]

**Pull request merged**